core/metrics/src/pusher.rs

```diff
@@ -44,7 +43,8 @@ pub async fn push_forever(host_url: &str) {
         }
     };
 
-    let mut push_interval = time::interval(Duration::from_secs(60));
+    let start = Instant::now() + Duration::from_secs(5);
```

Comment:
This is to wait until Services are initialized. 
Without this initial metrics push returns error, as metrics are empty and server does not like it.

Reply:
Can we skip pushing empty metrics instead?

Reply:
yes, but it is more tricky, since string with metrics is not empty (have some comment).
For the sake of simplicity I've dealt with it like that (not to employ flaky heuristics)